sgkit_plink/tests/data/plink_sim_10s_100v_10pmiss.bim

```diff
@@ -1,100 +1,100 @@
-1	1:1:A:C	0.0	1	C	A
-1	1:2:A:C	0.0	2	C	A
-1	1:3:A:C	0.0	3	C	A
-1	1:4:A:C	0.0	4	C	A
-1	1:5:A:C	0.0	5	C	A
-1	1:6:A:C	0.0	6	C	A
-1	1:7:A:C	0.0	7	C	A
-1	1:8:A:C	0.0	8	C	A
-1	1:9:A:C	0.0	9	C	A
+1	1:1:G:CGCGCG	0.0	1	CGCGCG	G
```

Comment:
Can you document how you generated this file?

Reply:
Hm how about this:

- I'll add some notes for now saying this was created using software in a separate environment (hail)
- Create an issue to document this properly and include the associated code that created it
- Wait to see where the multi-repo conversation goes (https://github.com/pystatgen/sgkit/issues/65#issuecomment-670049733)
- Add this to the `validation` folder if we merge repos since it is essentially the same process as the one I used in the REGENIE and HWE PRs

That sound good?  I'd rather not bootstrap another `validation`-like concept with all the associated CI changes if I can help it.

Reply:
@tomwhite does @eric-czech's proposal sound good?

Reply:
Yes (sorry forgot to reply) - sounds great!

Reply:
https://github.com/pystatgen/sgkit-plink/issues/29

Reply:
> I'll add some notes for now saying this was created using software in a separate environment (hail)

https://github.com/pystatgen/sgkit-plink/pull/20/commits/467d4bde5a645857b55c45176ac07e4b6c77e21c#diff-33e9afa89862f603b25f9c5abf5ef334R6